website/content/en/docs/cli/ansible-helm/_index.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Legacy CLI
+weight: 1
+description: Working with operator-sdk cli for ansible/helm based operators 
+---
+
+**Note:** For Golang Operators this CLI has been deprecated. Please consult the [new CLI reference][new_CLI] doc.
+
+[new_CLI]:/docs/cli/new-cli
```

Comment:
Incorrect link:
```suggestion
[new_CLI]:/docs/new-cli
```

We might want to wait until #3455 is merged to simplify this PR

Reply:
Never mind, we can merge #3455 after this one is merged.

Reply:
@estroz yea that'd be appreciated this PR has been in the works for a week now. 
Although, CI job request is still pending for this one.